### ISSUE TITLE
About the WER and CER

### ISSUE THREAD

**Issue by guoyingying432** [OlaWod/FreeVC#37]
Sorry to bother you, in your paper you mentioned that Word error rate (WER) and character error rate (CER) between source and converted speech are obtained by an ASR model. about the code for wer and cer calculation. Are you able to open source it?

**Comment by OlaWod** [comment]
Yes. You can find it [here](https://github.com/OlaWod/FreeVC/issues/9#issuecomment-1338658383).
